plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -30,6 +30,7 @@ import transformer, {
   rewriteDocLinks,
   addLinkClickListener,
   removeMkdocsHeader,
+  removeMkdocsCopyright,
```

Comment:
For sanity, I don't want this to mean _remove copyright_ and that sounds bad. (Plus we are not doing that anyway).
I suggest `removeMkdocsFooter`.

Reply:
OK. I'm fine with that. I chose _removeMKdocsCopyright_ because that was part of the name of the <div> element. "md-footer" is a parent of "md-footer-copyright" and it includes the "next page" link. I thought that _removeMkdocsFooter_ was a bit misleading. What do you think ? What about removeMkdocsCopyrightFooter ? Tell me what you prefer and I will do the renaming.

Reply:
One more suggestion in the mix: `simplifyMkDocsFooter`? :) Also known as weasel words

Reply:
Thanks @freben ! :D I like it. :)
Let's go for `simplifyMkdocsFooter`. In future we might want to add a new footer and so, remove will not make sense.

Reply:
I like it too. Let’s go with it. I’ll push an update this evening.